packages/functions/src/coordinator/CoordinatorBookAppointmentHandler.ts

```diff
@@ -19,6 +19,7 @@ export default async function (
   return bookAppointment(
     request.donorId,
     request.appointmentIds,
+    true,
```

Comment:
Should this be false?

Reply:
Coordinator appointments currently trigger a notification.
Should I disable it?

Should we have a centralized NotificationHandler to receive relevant information 
and a NotificationType enum, instead of a Boolean?

Reply:
BookAppointment is sufficiently complex in my opinion (70 rows),
to not write twice.
CancelAppointment was removed.